legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/guice/AbstractBaseModule.java

```diff
@@ -218,6 +220,15 @@ private ProjectStructureConfiguration getProjectStructureConfiguration()
         return (projectStructureConfiguration == null) ? ProjectStructureConfiguration.emptyConfiguration() : projectStructureConfiguration;
     }
 
+    private ProjectStructurePlatformExtensions getProjectStructurePlatformExtensions()
+    {
+        if (this.getProjectStructureConfiguration().getProjectPlatformsConfiguration() == null)
+        {
+            return null;
+        }
+        return this.getProjectStructureConfiguration().getProjectPlatformsConfiguration().buildProjectStructurePlatformExtensions();
+    }
```

Comment:
Let's only compute this once and store it in a field, similar to getProjectStructureExtensionProvider.